pyaedt/maxwell.py

```diff
@@ -1946,43 +1946,38 @@ def edit_external_circuit(self, netlist_file_path, schematic_design_name):
     @pyaedt_function_handler()
     def create_setup(self, setupname="MySetupAuto", setuptype=None, **kwargs):
         """Create an analysis setup for Maxwell 3D or 2D.
-        Optional arguments are passed along with ``setuptype`` and ``setupname``.  Keyword
-        names correspond to the ``setuptype``
-        corresponding to the native AEDT API.  The list of
-        keywords here is not exhaustive.
-
-        .. note::
-           This method overrides the ``Analysis.setup()`` method for the HFSS app.
-
-        Parameters
-        ----------
-        setuptype : int, str, optional
-            Type of the setup. Based on the solution type, options are
-            ``"HFSSDrivenAuto"``, ``"HFSSDrivenDefault"``,
-            ``"HFSSEigen"``, ``"HFSSTransient"`` and ``"HFSSSBR"``.
-            The default is ``"HFSSDrivenAuto"``.
-        setupname : str, optional
-            Name of the setup. The default is ``"Setup1"``.
-        **kwargs : dict, optional
-            Available keys depend on the setup chosen.
-            For more information, see :doc:`../SetupTemplatesMaxwell`.
-
-        Returns
-        -------
-        :class:`pyaedt.modules.SolveSetup.SetupMaxwell`
-            3D Solver Setup object.
-
-        References
-        ----------
-
-        >>> oModule.InsertSetup
-
-        Examples
-        --------
-
-        >>> from pyaedt import Maxwell3d
-        >>> app = Maxwell3d()
-        >>> app.create_setup(setupname="Setup1", setuptype="EddyCurrent", MaximumPasses=10,PercentError=2 )
+                Optional arguments are passed along with ``setuptype`` and ``setupname``.  Keyword
+                names correspond to the ``setuptype``
+                corresponding to the native AEDT API.  The list of
+                keywords here is not exhaustive.
+        ``
```

Comment:
```suggestion
        
```

Reply:
In addition, all the docstrings have an extra indentation, please check others' methods, and you will see how the indentation should be. Thanks